lib/linearize.js

```diff
@@ -217,9 +217,10 @@ module.exports = class LinearizedView extends EventEmitter {
   }
 
   _onOutputTruncated (output, length, forkId) {
+    // XXX: is forkId needed here ?
```

Comment:
Nah the length should suffice -- mind removing that?

Reply:
i removed